opdevnet/wallet/bech32.js

```
@@ -0,0 +1,173 @@
+// This file contains the implementation of the bech32 encoding and decoding functions.
+// This file was copied from the bech32 npm package with some slight modifications to work in the browser.
+// see https://cdn.jsdelivr.net/npm/bech32@2.0.0/dist/index.min.js
+
+'use strict';
```

Biome

[error] Redundant use strict directive.

The entire contents of JavaScript modules are automatically in strict mode, with no statement needed to initiate it.
Safe fix: Remove the redundant use strict directive.


(lint/suspicious/noRedundantUseStrict)